utility/get_ip_addrs.py

```diff
@@ -0,0 +1,57 @@
+""" Look at Ceph storage pods, return ip-addresses """
+import subprocess
+import shlex
+
+
+def get_ip_addrs_for_ceph_type(pod_type, namespace='openshift-storage'):
+    """
+    Get the ip address for all pods of a given type.
+    Args:
+        pod_type (string): pod type (mon, mgr, osd, all for everbody)
+        namespace (string): namespace in -n field of oc commands
+    Returns:
+        List of ip_addresses for all pods of the type specified
+    """
+    ret_list = []
+    look_for = f"rook-ceph-{pod_type}"
+    ipaddrs = get_ip_addrs(namespace)
+    for ceph_pod in ipaddrs:
+        if ceph_pod[0].startswith(look_for):
+            ret_list.append(ceph_pod[1])
+    return ret_list
+
+
+def get_ip_addrs(namespace='openshift-storage'):
+    """
+    Get the ip addresses of all pods in the namespaces.
+    Args:
+        namespace (string): namespace in -n field of oc commands
+    Returns:
+        List of two item lists of the form [pod_name, ip_address]
```

Comment:
Instead of list of lists how about returning a list of dicts [{pod_name:ip_addr}] ?

Reply:
Wouldn't be sufficient to return simple dict?
```
{
  'pod_name1': 'ip_addr1',
  'pod_name2': 'ip_addr2',
  'pod_name3': 'ip_addr3',
  ...
}
```

Reply:
+1 on @dahorak comment

Reply:
Using dahorak's suggestion.